src/YouTrackSharp/TimeTracking/TimeSettings.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace YouTrackSharp.TimeTracking
+{
+	/// <summary>
+	/// A class that represents YouTrack time settings information.
+	/// </summary>
+	public class TimeSettings
```

Comment:
Should we consider renaming to `TimeTrackingSettings`? (to stay close to the REST API description)
I see this is used for the "global" settings so perhaps `SystemWideTimeTrackingSettings` ?